src/main/java/jenkins/scm/api/mixin/ChangeRequest.java

```diff
@@ -0,0 +1,20 @@
+package jenkins.scm.api.mixin;
+
+import org.kohsuke.stapler.export.Exported;
+
+import javax.annotation.Nonnull;
+
+/**
+ * The interface carry with the information of a change request,
+ *  such as pull request or merge request.
+ * @since TODO
+ */
+public interface ChangeRequest {
```

Comment:
It is unclear to me what this interface is for. It does not seem to be referenced elsewhere in this PR. Is there some downstream PR or set of PRs which would use it somehow?